bootloader/grub.cfg

```diff
@@ -1,59 +1,44 @@
 set default="0"
-
-function load_video {
-  insmod efi_gop
-  insmod efi_uga
-  insmod video_bochs
-  insmod video_cirrus
-  insmod all_video
-}
-
-load_video
-set gfxpayload=keep
-insmod gzio
-insmod part_gpt
-insmod ext2
-
 set timeout=5
 
 menuentry "install" {
-    multiboot2 /boot/xen.efi dom0_max_vcpus=1-16 dom0_mem=max:8192M com1=115200,8n1 console=com1,vga
-    module2 /boot/vmlinuz console=hvc0 console=tty0
-    module2 /install.img
+    xen_hypervisor /boot/xen.efi dom0_max_vcpus=1-16 dom0_mem=max:8192M com1=115200,8n1 console=com1,vga
+    xen_module --name kernel /boot/vmlinuz console=hvc0 console=tty0
```

Comment:
Like the comment in the other ticket, I would remove the `--name`, more compatible with other configurations.